spec/controllers/users_controller_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+RSpec.describe UsersController, type: :controller do
+
+  describe "GET #show" do
+    it "returns http success" do
+      get :show
+      expect(response).to have_http_status(:success)
```

Comment:
Yes, it's giving me a 302 response which I know is a redirect. I'm all
ready testing that in the test below that one so I obviously don't need to
be testing the same thing twice right?
On Tue, Jan 12, 2016 at 12:25 PM bjblock <notifications@github.com> wrote:

> In spec/controllers/users_controller_spec.rb
> <https://github.com/tuckermullen/blocitoff/pull/2#discussion_r49508268>:
>
> > @@ -0,0 +1,16 @@
> > +require 'rails_helper'
> > +
> > +RSpec.describe UsersController, type: :controller do
> > +
> > +  describe "GET #show" do
> > +    it "returns http success" do
> > +      get :show
> > +      expect(response).to have_http_status(:success)
>
> I'm guessing this is the test that's failing?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tuckermullen/blocitoff/pull/2/files#r49508268>.
>
-- 

*Tucker Mullen*
253-208-9095